training/finetune_acft_hf_nlp/environments/acpt/context/requirements.txt

```diff
@@ -0,0 +1,12 @@
+azureml-acft-accelerator==0.0.2
+azureml-acft-contrib-hf-nlp==0.0.2
+mltable==1.1.0
+datasets==2.3.2
+mpi4py==3.1.3
+accelerate==0.11.0
+optimum==1.6.4
+transformers==4.26.1
+onnxruntime-training==1.14.1
+rouge-score==0.1.2
+sacrebleu==2.2.1
+azure-ai-ml==1.0.0
```

Comment:
You can also use template tags here to pin to the latest PyPI version: https://github.com/Azure/azureml-assets/wiki/Environments#template-tags

Reply:
We benchmarked our runs with v1.0.0; we can upgrade this package as part of next release

Reply:
[Version 1.0.0 is from October and a lot of bugs have been fixed since then](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ml/azure-ai-ml/CHANGELOG.md#100-2022-10-10). Please consider upgrading.